app/server/views/index.py

```diff
@@ -52,8 +61,13 @@ def upload():
 def deprecatedVendor():
     return render_template('index.html', js_bundle_main = get_js_bundle_filename())
 
+
```

Comment:
to be honest, I think this should all be handled client-side.

right now if you go to a production website `https://dev.withsempo.com/login` it doesn't actually load the bundle and returns a 404 😬

Reply:
Fixed here https://github.com/teamsempo/SempoBlockchain/pull/189